.gitignore

```diff
@@ -8,4 +8,5 @@
 *.pyc
 .vscode
 package-lock.json
```

Comment:
dont' you want to check in the package-lock.json now?

Reply:
Nope. With developers using separate package managers, the locks would often be ignored. Checking in the lock files proved more of a headache than security it gives over our dev-dependencies.

Reply:
@paulirish package-lock.json is ignored on the `npm install puppeteer` step; on the trybots, we'd like to install the same way our users do.